packages/trader/src/App/Components/Elements/PositionsDrawer/positions-drawer-card.jsx

```diff
@@ -169,10 +170,11 @@ const PositionsDrawerCard = ({
             <ResultOverlay
                 contract_id={id}
                 is_unsupported={is_unsupported}
+                is_visible={!!(contract_info.is_sold)}
                 has_same_contract_mounted={id === parseInt(active_position)}
                 onClickRemove={onClickRemove}
                 onClick={() => toggleUnsupportedContractModal(true)}
-                result={result}
+                result={(result || fallback_result)}
```

Comment:
why don't we have result?

Reply:
I couldn't reproduce it again (happened only once few weeks ago) but I'm guessing the `status` was returned wrongly which caused result to not return properly.
We're supposed to be getting `result` here on [L171](https://github.com/binary-com/deriv-app/blob/dev/packages/trader/src/Stores/Modules/Portfolio/portfolio-store.js#L171).

Reply:
I guess better to handle API failure, but ideally we shouldn't. let's keep it for now then

Reply:
Yeah, but I can't think of any other way since it's difficult to reproduce. 😅